pkg/sqlstore/impl/system/internal/db/id.sql.go

```diff
@@ -0,0 +1,24 @@
+package db
+
+import (
+	"context"
+)
+
+const (
+	getId    = `SELECT id FROM system_id`
+	insertId = `INSERT INTO system_id VALUES (?1)`
```

Comment:
Maybe in the future we'll discover that it's worth having a more general table like `validator_config` and use it as a key-value store. I suspect that will avoid having to create a new table for some new info we need to store.

We can do a migration if that's the case, so fine with this.